index.d.ts

```diff
@@ -442,8 +442,14 @@ declare namespace Moleculer {
 		error?: (ctx: Context<any, any>, err: Error) => Promise<void> | void;
 	}
 
+	interface RestSchema{
+		path: string,
+		method: 'GET' | 'POST' | 'DELETE' | 'PUT' | 'PATCH'
+	}
+
 	interface ActionSchema {
 		name?: string;
+		rest?: RestSchema,
```

Comment:
The param can also be `string`. For example
```js
list: {
            rest: "GET /",
            handler(ctx) {}
        }
```

Reply:
wouldn't it be more interesting to use an object with a method and optional url?

Reply:
because, have a path 

```javascript
  rest:{
         method: "GET", // here 
         path:"/" // here
                    },
            handler(ctx) {}
```        
that way we respect the idea of ​​typing

Reply:
Since `rest` can be a string (a.k.a. shorthand notation) it must be present in `index.d.ts`

Reply:
I got it So, I could use ``` rest ```  or the simplified form.

Reply:
@AndreMaz I alter for use string or rest object :)

```javascript
interface ActionSchema {
		name?: string;
		rest?: RestSchema | string,
```